### DISCUSSION TITLE
I just submitted Amazing.Module.HeroSlides to the Oqtane MarketPlace

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3084] [Q&A] [answered]
@sbwalker I don't see my module in the list does it go through a approval process

**Comment by sbwalker** [comment, marked as answer]
@vnetonline you have indicated in your module Release that it is only Compatible with Oqtane 1.0:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/196bd4eb-abff-4ca7-93fa-085ee629ca87)

**Reply by vnetonline** [reply]
Thanks @sbwalker
